crates/core/tedge/src/cli/mod.rs

```diff
@@ -36,6 +36,7 @@ mod upload;
     multicall(true),
 )]
 pub enum TEdgeOptMulticall {
+    /// Command line interface to interact with thin-edge.io
```

Comment:
The docs have to be updated too, to be in sync with the updated `tedge help`

Reply:
See https://github.com/thin-edge/thin-edge.io/pull/3313/commits/6a61c483664967daa102bc216fc4a889c9fffbbd